src/GitHub.InlineReviews/Views/InlineCommentPeekView.xaml

```diff
@@ -104,9 +104,11 @@
         <ScrollViewer Name="threadScroller"
                       VerticalScrollBarVisibility="Auto"
                       Background="{DynamicResource GitHubPeekViewBackground}">
-            <Grid>
-                <ghfvs:CommentThreadView x:Name="threadView" DataContext="{Binding Thread}"/>
-            </Grid>
+            <StackPanel Orientation="Vertical">
+                <Grid>
```

Comment:
Why are a `StackPanel` and `a  Grid` here when there is only one child? Are these necessary?

Reply:
@donokuda Did you do this or did I?

Reply:
This might have been me. 😅 I think it should be okay to remove the `<Grid>`
I'm not even sure if there's a benefit specifying the `Orientation="Vertical"` since there's only one child. Maybe that can be 🔥'd too.

Reply:
👍